l5kit/l5kit/tests/conftest.py

```diff
@@ -29,7 +33,21 @@ def cfg() -> dict:
 
 
 @pytest.fixture(scope="session")
-def zarr_dataset() -> ChunkedDataset:
-    zarr_dataset = ChunkedDataset(path="./l5kit/tests/artefacts/single_scene.zarr")
+def zarr_dataset(dmg: LocalDataManager) -> ChunkedDataset:
+    zarr_path = dmg.require("single_scene.zarr")
```

Comment:
Are we making sure that the 'single_scene.zarr' file is shipped even in the pip package?

Reply:
tests are not shipped at all in the package

Reply:
ahh then that's fine!